unliker.py

```diff
@@ -42,9 +42,8 @@ def process_like_page(client, date_threshold, next_max_id=None):
     process_next_page = False
     has_next_page = liked['more_available'] and next_max_id is not None
     for item in liked["items"]:
-        if is_suspicious(item):
+        if is_suspicious(item) and get_taken_at_timestamp(item) > date_threshold:
             suspicious_likes.append(item)
-        if get_taken_at_timestamp(item) > date_threshold:
```

Comment:
I agree with the above additional check, but please make the check 
```
if get_taken_at_timestamp(item) > date_threshold:
    process_next_page = has_next_page
```
separate, as it was. There is a possibility that one page has no suspicious likes (you recently liked more than 20 items), but the next page has. If you don't set `process_next_page = has_next_page` when you don't find any suspicious like on a page, the loop will break.

Reply:
I realized that after, I'm working on another pull, I figured I'd move the date checking logic inside `is_suspicious` itself, but to avoid redundancy we can check for `is_suspicious` inside the scope of ` if get_taken_at_timestamp(item) > date_threshold:`

Reply:
+sprinkled some pep8 blessings